src/content/guides/asset-management.md

```diff
@@ -6,13 +6,14 @@ contributors:
   - michael-ciniawsky
   - TheDutchCoder
   - sudarsangp
+  - chrischinchilla
 ---
 
-If you've been following the guides from the start, you will now have a small project that shows "Hello webpack". Now let's try to incorporate some other assets, like images, to see how they can be handled.
+If you've been following the guides from the start, you will now have a small project that shows "Hello webpack". Now let's try to incorporate some other assets, like images, to see how webpack handles them.
 
-Prior to webpack, front-end developers would use tools like grunt and gulp to process these assets and move them from their `/src` folder into their `/dist` or `/build` directory. The same idea was used for JavaScript modules, but tools like webpack will __dynamically bundle__ all dependencies (creating what's known as a [dependency graph](/concepts/dependency-graph)). This is great because every module now _explicitly states its dependencies_ and we'll avoid bundling modules that aren't in use.
+Prior to webpack, front-end developers used tools like grunt and gulp to process these assets and move them from their `/src` folder into their `/dist` or `/build` directory. They used the same idea for JavaScript modules, but tools like webpack __dynamically bundle__ all dependencies (creating what's known as a [dependency graph](/concepts/dependency-graph)). This is means that every module now _explicitly states its dependencies_ and we avoid bundling modules that aren't in use.
 
-One of the coolest webpack features is that you can also _include any other type of file_, besides JavaScript, for which there is a loader. This means that the same benefits listed above for JavaScript (e.g. explicit dependencies) can be applied to everything used in building a website or web app. Let's start with CSS, as you may already be familiar with that setup.
+One of the coolest webpack features is that we can also _include any other type of file_, besides JavaScript, for which there is a loader. This means that the same benefits listed above for JavaScript (e.g. explicit dependencies) apply to everything used in building a website or web app. Let's start with CSS, as you may already be familiar with that setup.
```

Comment:
Would 'it' be a better use here to refer to webpack? I don't really like the 'we' form here.

Reply:
I'm using 'we' here because it's used everywhere else, neither 'you' or 'we' is wrong or right, but you should pick one and stick to it, this document was majority 'we', so… But in this case, yes, 'it' is more correct.

Throughout most of the docs there's fairly inconsistent use of the person, but that would be a lot more work and involve some further thought…

Reply:
We are saying _one of the coolest_ a lot in this content, I suggest changing to something else and more formal. Like _In webpack you can also include any type of file_.